modal/secret.py

```diff
@@ -38,4 +39,19 @@ async def _load(resolver: Resolver) -> _SecretHandle:
         super().__init__(_load, rep)
 
 
+async def _resolve_secret(resolver: Resolver, secret: _Secret) -> str:
```

Comment:
Your earlier comment shows up to me. I asked:

To clarify: are you asking to move the method `_resolve_secret` to [object.py](https://github.com/modal-labs/modal-client/blob/main/modal/object.py#L105) (as in a simple namespace change) or to add it / integrate it to the `Handle.from_app` method?

Reply:
Actually looking at the code, I realize we just raise an error message from the server, so it might make sense to just put this in the server instead. I'm happy to do that – I'm working on exactly that code right now.

Reply:
Ok sounds good.